### ISSUE TITLE
General editorial comments on the WCAG2ICT draft

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#1001]
- Be sure that when the document shows verbiage replacements the quotation marks are correct and contain all of the language that replaced the word or phrase from WCAG. 
    - [ ] SC 2.1.2 the verbiage replacement is actually "non-web document or non-web software" - don't need the extra quotes in the middle because that whole phrase is replacing the one word "page".
    - [ ] SC 2.2.1 - same word replacement issue plus there is an extra space before the word content that needs removing so it reads: "content" with "non-web document or non-web software".
    - [ ] SC 2.2.2 - same word replacement issue because you're replacing one word "page" with the phrase.   Plus the word replacement in Note 2 is actually replacing "on the web page with the whole phrase "on the web page or in the non-web document". Plus the replacement markup in Note 2 should include the word "or" because it didn't exist in the original WCAG Note 2.
    - [ ] SC 2.4.5 - the description of the word replacement is incorrect (for non-web software) and doesn't include all the words in the quote. It should be "non-web document within a set of non-web documents, or a non-web software program within a set of non-web software programs".
    - [ ] SC 3.1.1 - the description of the word replacement should be "non-web documents or non-web software"
    - [ ] SC 3.1.2 - the description of the word replacement should be "non-web documents or non-web software" 
    - [ ] Guideline 3.2 - the description of the word replacement should be "non-web documents or non-web software"
    - [ ] SC 3.2.4 - the description of the phrase replacement should be “set of non-web documents or set of non-web software programs”
    - [ ] SC 3.3.4 - same comment - the description of the phrase replacement should be “set of non-web documents or set of non-web software programs”
    - [ ] SC 3.3.8 - A tiny mismatch - Word replacement says... replacing “the website” with “a non-web document or non-web software” but in the list where the replacement is used in Personal Content there's a comma after "a non-web document". So either remove the comma or add it to the word replacement text.
    - [ ] 2.4.8 - Missing "a" in the word replacement description so it reads "set of non-web documents or a set of non-web software programs".
